cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -71,16 +72,8 @@ coverage: clean
 isort_check: ## check that isort has been run
 	isort --check-only {{cookiecutter.project_name}}/
 
-isort: ## run isort to sort imports in all Python files
-	isort --recursive --atomic {{cookiecutter.project_name}}/
-
-style: ## run Python style checker
-	pylint --rcfile=pylintrc {{cookiecutter.project_name}} *.py
-
-lint: ## run Python code linting
-	pylint --rcfile=pylintrc {{cookiecutter.project_name}} *.py
-
-quality: style isort_check lint ## check code style and import sorting, then lint
+quality:
```

Comment:
What I am getting from all these removed commands and new `quality` target is that all these commands will be executed within the `tox` command but what if we still want to run let say just lint command then I wouldn't have any make target for that. We can still keep the old make targets intact in the file and just update the quality check to run tox.

Reply:
updated the targets as per the suggestion.